osvr/RenderKit/osvr_display_configuration.h

```diff
@@ -132,6 +132,9 @@ class OSVRDisplayConfiguration {
     std::vector<float> const& getDistortionPolynomalBlue() const;
     ///@}
 
+    /// Returns the desired number of triangles in the constructed distortion mesh.
+    int OSVR_RENDERMANAGER_EXPORT getDesiredDistortionTriangleCount() const;
```

Comment:
This code follows the rest of the entries that are already in that section of the config file, which use the underscore rather than camelCase.  Making this one different would be locally inconsistent.  Changing all of them will break existing configuration files.